src/main/controllers/SubscriptionCaseSearchController.ts

```diff
@@ -0,0 +1,36 @@
+import { Response } from 'express';
+import {HearingRequests} from '../resources/requests/hearingRequests';
+import {cloneDeep} from 'lodash';
+import {PipRequest} from '../models/request/PipRequest';
+
+const hearingRequests = new HearingRequests();
+
+export default class SubscriptionCaseSearchController {
+
+  public async get(req: PipRequest, res: Response):  Promise<void> {
```

Comment:
```suggestion
  public get(req: PipRequest, res: Response): void {
```
try not to randomly use async when its not needed